frontend/src/components/pages/acls/PrincipalGroupEditor.tsx

```diff
@@ -253,17 +256,19 @@ export const AclPrincipalGroupEditor = observer((p: {
 
 const ResourceACLsEditor = observer((p: {
     resource: ResourceACLs,
+    resourceType: AclStrResourceType,
     onDelete?: () => void
 }) => {
     const res = p.resource;
     const isCluster = !('selector' in res);
-    const isTopic = ('Write' in res.permissions);
-    const isConsumerGroup = !isCluster && !isTopic;
     const isAllSet = res.all == 'Allow' || res.all == 'Deny';
 
     let resourceName = 'Cluster';
-    if (isTopic) resourceName = 'Topic';
-    if (isConsumerGroup) resourceName = 'Consumer Group';
+    if (p.resourceType == 'Topic') resourceName = 'Topic';
+    if (p.resourceType == 'Group') resourceName = 'Consumer Group';
+    if (p.resourceType == 'TransactionalID') resourceName = 'Transactional ID';
+
```

Comment:
Can this be any other types?  Just wondering if there's a fallback.

Reply:
acl type is:
``` 

export type AclStrResourceType =
    | 'Unknown'
    | 'Any'
    | 'Topic'
    | 'Group'
    | 'Cluster'
    | 'TransactionalID'
    | 'DelegationToken'
    ;

```

Unknown: should never be reported by the backend.
Any: is only for search.
DelegationToken: is not supported.

By default it shows Cluster as name.